resources/lib/games.py

```diff
@@ -458,122 +462,63 @@ def BROWSE_YEARS(plugin, year, team=False):
+                                PLAY_OPTIONS_URL % gameID,
                                 headers=headers,
-                                max_age=0
+                                max_age=cache_max_age
                             ).json()
-    url = Response['path']
-    drm = Response['drmToken']
-    try:
-        stream_type = Response['streamType']
-        if stream_type == 'dash':
-            protocol = 'mpd'
-        else:
-            protocol = 'hls'
-    except:
-        protocol = 'hls'
+    for play_option in play_options['Vods']:
+        yield Listitem.from_dict(
+                                PLAY_GAME,
+                                bold(play_option['DisplayName'][0]['Value']),
+                                params = {
+                                    'gameID': gameID,
+                                    'videoProfileId': play_option['PlayActions'][0]['VideoProfile']['Id']
+                                }
+                            )
 
-    headers = {'User-Agent': USER_AGENT}
-    start_point = None
-    live_play_type = int(Script.setting.get_string('live_play_type'))
-    ret = None
-    if game_type == 'live':
-        if live_play_type == 0:
-            line1 = "Start from Beginning"
-            line2 = "Go LIVE"
-            ret = xbmcgui.Dialog().select("Game Options", [line1, line2])
-            if ret == -1:
-                yield None
-                return
-        if ret == 0 or live_play_type == 2:
-            url = url.replace('br_long_master', 'master')
-        content = urlquick.get(url, headers=headers).text
-        sample = re.findall('(.*video.*\.m3u8?)', content)[0]
-        match = re.search('(https?)://([^:]+)/([^?]+?)\?(.+)$', url)
-        baseurl = os.path.dirname(match.group(1)+'://'+match.group(2)+'/'+match.group(3))
-        ql_url = baseurl + '/' + sample
-        content = urlquick.get(ql_url, headers=headers, max_age=0).text
-        durations = re.findall('\#EXTINF\:([0-9]+\.[0-9]+)\,', content)
-        duration = sum([float(i) for i in durations])
-        if ret == 0 or live_play_type == 2:
-            stream_start = re.findall('PROGRAM\-DATE\-TIME\:(.*)', content)[0]
-            stream_start = time.strptime(stream_start, '%Y-%m-%dT%H:%M:%S.%fZ')
-            stream_start_ts = calendar.timegm(stream_start) * 1000
-            start_point = str(int((start_time - stream_start_ts) / 1000))
-        elif ret == 1 or live_play_type == 1 :
-            start_point = str(duration).split('.')[0]
 
+@Resolver.register
+def PLAY_GAME(plugin, gameID, videoProfileId):
+    access_token = get_token()
+    deviceinfos = get_device_ids()
+    headers = {
+        'content-type': 'text/plain;charset=UTF-8',
+        'authorizationtoken': access_token,
+        'azukiimc': 'IMC7.1.0_AN_D3.0.0_S0',
+        'deviceprofile': b64encode(('{"osVersion":"10","vendorName":"Microsoft","osName":"HTML5","deviceUUID":"%s"}' % deviceinfos['DEVICEID']).encode('ascii'))
```

Comment:
This line seems pretty Windows-specific. Is it here to "spoof" a browser/OS combination to the API? Or should we rather report the actual OS version?

Reply:
I tried to spoof the API, tried different `deviceprofiles` but did not work

Reply:
@mkorganashvili Interesting. So you're saying that it _only_ works with the above `deviceprofile`?

Reply:
I mean it works with any deviceprofile